### PR TITLE
Fix #2, add react-native-web dependency and update App.js

### LINKED ISSUE #2: Can't resolve 'react-native-web/dist/exports/AppRegistry'
```
Module not found: Can't resolve 'react-native-web/dist/exports/AppRegistry' 
in '/tmp/expo-template-scala/node_modules/expo/build/environment'
```

I get this error, when I try run the app.

` expo start --web`

`yarn add react-native-web react-dom expo`

```
 web  Failed to compile.
/tmp/expo-template-scala/node_modules/expo/build/Notifications/Notifications.js
Module not found: Can't resolve 'react-native-web/dist/exports/AsyncStorage' in '/tmp/expo-template-scala/node_modules/expo/build/Notifications'
```

### PR DESCRIPTION
**Pull request by manuzhang**
As per [solutions](https://github.com/shadaj/expo-template-scala/issues/2#issuecomment-669303248) provided by @thesamet

### CODE REVIEW

**Comment by manuzhang** [comment]
@shadaj, I've only tested with local project created from this template so I'm not sure it works for new project. Please let me know if there is a way to test it.

File: package.json [hunk 1]
```diff
@@ -16,6 +16,7 @@
     "react": "16.9.0",
     "react-dom": "16.9.0",
     "react-native": "https://github.com/expo/react-native/archive/sdk-37.0.1.tar.gz",
+		"react-native-web": "^0.13.14",
```
Comment (shadaj) [inline package.json]: The indentation seems off here

**Review by shadaj — CHANGES_REQUESTED** [review]
Just one nit, but otherwise LGTM! We don't have much of a testing process for this template, checking with a local project has been the way I test as well so we should be good to go ahead with the changes.

**Comment by manuzhang** [comment]
Thanks. Indent fixed.

**Review by shadaj — APPROVED** [review]

**Pull request merged**